packages/cardpay-sdk/sdk/prepaid-card/base.ts

```diff
@@ -388,22 +434,39 @@ export default class PrepaidCard {
     };
   }
 
+  async create(txnHash: string): Promise<{ prepaidCards: PrepaidCardSafe[]; txnReceipt: TransactionReceipt }>;
   async create(
     safeAddress: string,
     tokenAddress: string,
     faceValues: number[],
     customizationDID: string | undefined,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<{ prepaidCards: PrepaidCardSafe[]; txnReceipt: TransactionReceipt }>;
+  async create(
+    safeAddressOrTxnHash: string,
+    tokenAddress?: string,
+    faceValues?: number[],
+    customizationDID?: string | undefined,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<{ prepaidCards: PrepaidCardSafe[]; txnReceipt: TransactionReceipt }> {
-    if (faceValues.length > MAX_PREPAID_CARD_AMOUNT) {
+    if (!tokenAddress) {
```

Comment:
maybe a better test is that `safeAddressOrTxnHash` actually looks like an address, that way you are asserting on the value you care about rather than making an assertion on a side effect. All addresses have 40 characters plus "0x" (so 42 characters in total). All txn hashes have 64 characters plus "0x" (so 66 chars in total).